### PR TITLE
fix toggle accessibility bug:

### PR DESCRIPTION
**Pull request by liuqingbo**
fix bugs:
1. when focusing on toggle, screen reader doesn't read current state.
2. when changing toggle value, screen reader doesn't read changed value.

### CODE REVIEW

File: src/components/Toggle/Toggle.tsx [hunk 1]
```diff
@@ -72,12 +72,22 @@ export class Toggle extends React.Component<IToggleProps, IToggleState> {
...
             />
             <div className='ms-Toggle-background'>
               <div className='ms-Toggle-focus' />
               <div className='ms-Toggle-thumb' />
             </div>
-            <Label className='ms-Toggle-stateText'>{ stateText }</Label>
+            <Label
+              className='ms-Toggle-stateText'
+              role='status'
+              aria-live='polite'
```
Comment (cschlechty) [inline src/components/Toggle/Toggle.tsx]: This is overkill and shouldn't be needed. There's only 2 states. The current state gets read, and activating the button flips it.

File: src/components/Toggle/Toggle.tsx [hunk 2]
```diff
@@ -72,12 +72,22 @@ export class Toggle extends React.Component<IToggleProps, IToggleState> {
               role='button'
               aria-pressed={ isChecked }
               onClick={ this._onClick }
+              aria-describedby={ this._id + 'ariaDescribedBy' }
```
Comment (cschlechty) [inline src/components/Toggle/Toggle.tsx]: Why is pressed alone not sufficient?

File: src/components/Toggle/Toggle.tsx [hunk 3]
```diff
@@ -72,12 +72,22 @@ export class Toggle extends React.Component<IToggleProps, IToggleState> {
...
-            <Label className='ms-Toggle-stateText'>{ stateText }</Label>
+            <Label
+              className='ms-Toggle-stateText'
+              role='status'
+              aria-live='polite'
+            >
+              { stateText }
+            </Label>
+            <p className='ms-u-screenReaderOnly' id={ this._id + 'ariaDescribedBy' }>
+              { 'current state is ' + (isChecked ? 'on' : 'off') }
```
Comment (antonlabunets) [inline src/components/Toggle/Toggle.tsx]: We also do NOT put any kind of strings (that need to be localized) into the implementation. If one is required - you should make it a prop.

File: src/components/Toggle/Toggle.tsx [hunk 4]
```diff
@@ -72,12 +72,22 @@ export class Toggle extends React.Component<IToggleProps, IToggleState> {
               role='button'
               aria-pressed={ isChecked }
```
Comment (antonlabunets) [inline src/components/Toggle/Toggle.tsx]: Even though the binding of pressed to checked looks right from the code perspective, when I listed to the example in JAWS - after a switch or two it started saying the opposite. It didn't matter if it was cursor mode (++) or not...

**Comment by antonlabunets** [comment]
I have a different and simpler fix at: https://github.com/OfficeDev/office-ui-fabric-react/pull/130

**Comment by dzearing** [comment]
Closing this as the other simpler fix was merged.

**Pull request closed without merging**